curve25519-dalek/src/edwards.rs

```diff
@@ -542,6 +546,14 @@ impl EdwardsPoint {
         }
     }
 
+    /// Convert to `AffinePoint`.
+    pub(crate) fn to_affine(self) -> AffinePoint {
```

Comment:
This can't possibly be a total function, right? The identity point doesn't have an affine representation.